src/components/developerCards/index.jsx

```diff
@@ -72,6 +73,7 @@ class DeveloperCard extends Component {
 
   renderCards = () => {
     const { data, openModal, changeModalTypes } = this.props;
+    console.log('api data ------>',data);
```

Comment:
Also, don't forget to delete all console logs when you are done with the changes you are working on